### DISCUSSION TITLE
API SuSo for Stata

### DISCUSSION THREAD

**Discussion by cae-ins** [arthur-shaw/susoapi#41] [General]
Morning, we are running LFS with Survey Solutions. Is it any SuSo API for Stata ?

**Comment by arthur-shaw** [comment]
Yes, there's a Stata package of the same name as my package. See the links [here](https://docs.mysurvey.solutions/headquarters/api/survey-solutions-api/)
for the SuSo team's compilation of API clients.

There's also another Stata package not listed in that compilation. [Here's](https://github.com/petbrueck/sursol) the repo.

But whatever the Stata package, it relies on another tool that can make/parse web requests made to SuSo's API endpoints, which means you'll have to install the dependency for the SuSo API package to work. susoapi relies on Python. sursol relies on R. Both provide Stata functions as the user interface.
